### PR TITLE
Add fedora-xfce and centos templates

### PR DESCRIPTION
**Pull request by nastaziya**

### CODE REVIEW

File: user/managing-os/templates.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ We guarantee that the binary updates are compiled from exactly the same source c
 
  * [Fedora](/doc/templates/fedora/) (default base template)
  * [Fedora - Minimal](/doc/templates/fedora-minimal)
+ * [Fedora - Xfce](/doc/templates/fedora-xfce)
```
Comment (andrewdavidwong) [inline user/managing-os/templates.md]: Is `Fedora - Xfce` really an official template?

CC: @marmarek

File: external/os-guides/centos/centos-minimal.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+---
+layout: doc
+title: CentOS Template
```
Comment (andrewdavidwong) [inline external/os-guides/centos/centos-minimal.md]: This should be `CentOS Minimal Template` to distinguish it from the full `CentOS Template`.

File: user/managing-os/templates.md [hunk 3]
```diff
@@ -43,6 +44,7 @@ By installing these templates, you are trusting not only the Qubes developers an
 * [Whonix](/doc/templates/whonix/)
 * [Ubuntu](/doc/templates/ubuntu/)
 * [Archlinux](/doc/templates/archlinux/)
+* [CentOS](/doc/templates/centos/)
```
Comment (andrewdavidwong) [inline user/managing-os/templates.md]: I think `CentOS Minimal` is missing from this list.

File: user/managing-os/fedora/fedora.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ The Fedora TemplateVM
 
 The Fedora [TemplateVM] is the default TemplateVM in Qubes OS.
 This page is about the standard (or "full") Fedora TemplateVM.
-For the minimal version, please see the [Fedora Minimal] page.
+For the minimal and Xfce version, please see the [Fedora Minimal] and [Fedora Xfce] page.
```
Comment (andrewdavidwong) [inline user/managing-os/fedora/fedora.md]: Minor: should now be `versions` instead of `version`.

File: user/managing-os/templates.md (hunk 1 above)
Comment (marmarek) [reply user/managing-os/templates.md]: I'm fine with adding it as such. It's based on official Fedora template and also is regularly tested by @fepitre

**Comment by nastaziya** [comment]
I've made all the changes requested by @andrewdavidwong as well as some readability adjustments suggested by @Grivel-l

**Comment by andrewdavidwong** [comment]
Thanks, @nastaziya. This is not essential, but it might be nice to briefly describe the practical user-facing difference between the GNOME and Xfce Fedora templates. Since VM windows are seamlessly displayed on the dom0 desktop by default, most of the user's exposure to traditional desktop environment elements is through dom0, making the choice of DE in templates ostensibly less important. Is the choice of template DE primarily a matter of which apps are installed by default (terminal emulator, file manager, etc.)?

**Comment by fepitre** [comment]
> Thanks, @nastaziya. This is not essential, but it might be nice to briefly describe the practical user-facing difference between the GNOME and Xfce Fedora templates. Since VM windows are seamlessly displayed on the dom0 desktop by default, most of the user's exposure to traditional desktop environment elements is through dom0, making the choice of DE in templates ostensibly less important. Is the choice of template DE primarily a matter of which apps are installed by default (terminal emulator, file manager, etc.)?

Yes it is. Initially, when I implemented XFCE flavor for Fedora and then in CentOS, it was motivated by having only XFCE apps installed.

**Comment by andrewdavidwong** [comment]
I've resolved the merge conflicts. @nastaziya, please note that we recently consolidated the "minimal" template pages into a single page:

https://www.qubes-os.org/doc/templates/minimal/

Most of the page was being duplicated every time someone added a new minimal template, which would have made maintaining the documentation extremely difficult. As you can see, the general sections of the new consolidated page apply to all minimal templates, and there is a section for distro-specific notes at the end. Could you please convert the CentOS Minimal page to this new structure? Thank you!

File: user/managing-os/fedora/fedora-xfce.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+If you would like to use Fedora Xfce (more lightweight compared to GNOME desktop environment) Linux distribution in your qubes, you can install one of the available Fedora Xfce templates.
+
+
+Installing
+----------
+
+To install a specific Fedora Xfce TemplateVM that is not currently installed in your system, use the following command in dom0:
+
+    $ sudo qubes-dom0-update --enablerepo=qubes-templates-itl-testing qubes-template-fedora-XX-xfce
```
Comment (marmarek) [inline user/managing-os/fedora/fedora-xfce.md]:
```suggestion
    $ sudo qubes-dom0-update --enablerepo=qubes-templates-itl qubes-template-fedora-XX-xfce
```

File: user/managing-os/minimal-templates.md [hunk 6]
```diff
@@ -162,9 +166,31 @@ Documentation on all of these can be found in the [docs](/doc)
...
+### CentOS
+
+As is the case with above-mentioned Minimal Templates, the required packages are to be installed in the running template with the following command (replace `packages` with a space-delimited list of packages to be installed):
+
+    [user@your-new-clone ~]$ sudo yum install packages
+
+Use case | Description | Required steps
+--- | --- | ---
+**Standard utilities** | If you need the commonly used utilities | Install the following packages: `pciutils` `vim-minimal` `less` `psmisc` `gnome-keyring`
+**Networking** | If you want networking | Install `qubes-core-agent-networking` `qubes-core-agent-network-manager` `NetworkManager-wifi` `network-manager-applet` `wireless-tools` `dejavu-sans-fonts` `notification-daemon` `gnome-keyring`
```
Comment (marmarek) [inline user/managing-os/minimal-templates.md]: How `dejavu-sans-fonts` is related to networking?

File: user/managing-os/minimal-templates.md [hunk 7]
```diff
@@ -162,9 +166,31 @@ Documentation on all of these can be found in the [docs](/doc)
...
+**Audio** | If you want sound from your VM... | Install `pulseaudio-qubes`
+**FirewallVM** | You can use the minimal template as a template for a [FirewallVM](/doc/firewall/), like `sys-firewall` | Install `qubes-core-agent-networking`, and `nftables`.  Also install `qubes-core-agent-dom0-updates`(script required to handle `dom0` updates), if you want to use a qube based on the template as an updateVM (normally sys-firewall).
```
Comment (marmarek) [inline user/managing-os/minimal-templates.md]: `nftables` is automatically installed by `qubes-core-agent-networking`, no need to name it explicitly.

File: user/managing-os/minimal-templates.md [hunk 8]
```diff
@@ -162,9 +166,31 @@ Documentation on all of these can be found in the [docs](/doc)
...
+**NetVM** | You can use this template as the basis for a NetVM such as `sys-net` | Install the following packages: `qubes-core-agent-networking`, `qubes-core-agent-network-manager` and `nftables`.
```
Comment (marmarek) [inline user/managing-os/minimal-templates.md]: Same here about `nftables`.

File: user/managing-os/minimal-templates.md [hunk 9]
```diff
@@ -162,9 +166,31 @@ Documentation on all of these can be found in the [docs](/doc)
...
+**VPN** | You can use this template as basis for a [VPN](/doc/vpn/) qube | You may need to install network-manager VPN packages, depending on the VPN technology you'll be using. After creating a machine based on this template, follow the [VPN howto](/doc/vpn/#set-up-a-proxyvm-as-a-vpn-gateway-using-networkmanager) to configure it.
+**Desktop environment** | To improve desktop experience using additional packages from the `qubes-core-agent` | `qubes-menus` which defines menu layout, `qubes-desktop-linux-common` which contains icons and scripts to improve desktop experience. `qubes-core-agent-nautilus`/`qubes-core-agent-thunar`: packages providing integration with the Nautilus/Thunar file manager (without it, items like "copy to VM/open in disposable VM" will not be shown in Nautilus/Thunar).
+**Additional services** | If you need additional Qubes services | Install `qubes-gpg-split` `qubes-pdf-converter` `qubes-img-converter`("Qubes apps" implementing split GPG, trusted PDF and image converter), `qubes-snapd-helper`(if you want to use snaps), `qubes-mgmt-\*`(if you want to use salt management on the template and qubes).
```
Comment (marmarek) [inline user/managing-os/minimal-templates.md]: As for salt, it should be enough to install `qubes-mgmt-salt-vm-connector` - it should pull required packages. Most of `qubes-mgmt-*` packages are needed in dom0 only.

File: user/managing-os/minimal-templates.md (hunk 6 above)
Comment (awokd) [reply user/managing-os/minimal-templates.md]: FWIW, `dejavu-sans-fonts` has been part of the Fedora minimal template doc. for a [long time](https://web.archive.org/web/20150424102809/https://www.qubes-os.org/doc/Templates/FedoraMinimal/). [This](https://www.qubes-os.org/doc/fedora-minimal-template-customization/#lightweight-packages-recommendations) says it is for "nicer GUI menus" (possibly in Network Manager?), but I don't know if it's still necessary.

Comment (nastaziya) [reply user/managing-os/minimal-templates.md]: Both Fedora and CentOS minimal comes with `dejavu-sans-fonts` already installed, so it's not necessary(removed from both templates doc).

**Pull request merged**